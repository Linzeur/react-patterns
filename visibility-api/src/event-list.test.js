import React from "react";
import { render } from "@testing-library/react";

import ListEvent from "./events-list";

const listEvents = [
  { date: new Date("06/15/2019"), value: false },
  { date: new Date("06/18/2019"), value: false },
  { date: new Date("06/19/2019"), value: true }
];
test("Test event list component", () => {
  const { asFragment } = render(
    <ListEvent title={"Titulo 1"} listEvents={listEvents} />
  );

  expect(asFragment()).toMatchSnapshot();
});
