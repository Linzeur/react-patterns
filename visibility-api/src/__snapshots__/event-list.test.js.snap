// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test event list component 1`] = `
<DocumentFragment>
  Titulo 1
  <ul>
    <li>
      Sat Jun 15 2019 00:00:00 GMT-0500 (Peru Standard Time), value of event: false
    </li>
    <li>
      Tue Jun 18 2019 00:00:00 GMT-0500 (Peru Standard Time), value of event: false
    </li>
    <li>
      Wed Jun 19 2019 00:00:00 GMT-0500 (Peru Standard Time), value of event: true
    </li>
  </ul>
</DocumentFragment>
`;
